index.html

```diff
@@ -287,5 +293,50 @@ <h3>Skills</h3>
         </ul>
       </div>
     </section>
+
+    <!-- CONTACT FORM -->
+    <div class="contact-container">
+      <div class="container">
+        <h2>Contact me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I'd love to help
+          with it!
+        </p>
+        <div class="contact-form">
+          <form action="https://formspree.io/f/mjvdabgg" method="POST">
+            <div class="form-group">
+              <input
+                type="text"
+                id="name"
+                placeholder="Enter your name"
+                name="name"
+                maxlength="30"
+                required
+              />
+            </div>
+            <div class="form-group">
+              <input
+                type="email"
+                id="email"
+                name="email"
+                placeholder="Enter your email"
+                required
+              />
+            </div>
+            <div class="form-group">
+              <textarea
+                id="message"
+                name="message"
+                placeholder="Write your message here..."
+                required
+                maxlength="500"
+              ></textarea>
+            </div>
+            <button type="submit">Get in touch</button>
+          </form>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Kindly make sure to wrap this section into a `section` tag in order to follow HTML best practices.

Reply:
Thanks for notifying me. I've wrapped it in a `section` tag.

Reply:
Fixed.